solr/solrj/src/java/org/apache/solr/client/solrj/impl/ZkClientClusterStateProvider.java

```diff
@@ -215,6 +215,11 @@ static String buildZkHostString(Collection<String> zkHosts, String chroot) {
     return zkHostString;
   }
 
+  @Override
+  public String getQuorumHosts() {
+    return zkStateReader.getZkClient().getZkServerAddress();
```

Comment:
This seems to hit a NPE in a bunch of the recent test failures.

Reply:
Even when I replace `zkStateReader` with `getZkStateReader()`, some tests fail.

I believe the `CloudHttp2SolrClientBuilderTest` was designed to simply test that the (dummy) ZK_HOST provided is being set, but the way the test was changed in this PR makes it want to connect to that ZK, which of course fails...